docs-kits/kits/Traceability Kit/page_architecture-view.mdx

```diff
@@ -85,10 +85,11 @@ The following diagram shows a basic data processing flow how a company's interna
 Data provisioning of Traceabilty is built on the data provisioning of the [Industry Core KIT](../Industry%20Core%20Kit/Architecture%20View%20Industry%20Core%20Kit), i.e., Traceability extends the digital twins PartType and PartInstance with additional aspect models:
 
 - Digital Twin "PartType"
-
+  - N/A
 - Digital Twin "PartInstance"
-  - Aspect model ["TractionBatteryCode"](../Traceability%20Kit/Software%20Development%20View/Data%20Provider%20Development%20View%20Traceability%20Kit#tractionbatterycode)
-
+  - Aspect model ["TractionBatteryCode"](https://eclipse-tractusx.github.io/docs-kits/kits/Traceability%20Kit/Software%20Development%20View/Data%20Provider%20Development%20View%20Traceability%20Kit#tractionbatterycode)
+  - Aspect model ["SoftwareInformation"](https://eclipse-tractusx.github.io/docs-kits/kits/Traceability%20Kit/Software%20Development%20View/Data%20Provider%20Development%20View%20Traceability%20Kit#softwareinformation)
+  - Aspect model ["CertificateSigningRequests"](https://eclipse-tractusx.github.io/docs-kits/kits/Traceability%20Kit/Software%20Development%20View/Data%20Provider%20Development%20View%20Traceability%20Kit#certificatesigningrequests)
```

Comment:
I'm curious about the Aspect Models "SoftwareInformation" and "CertificateSigningRequests". These don't appear to be a part of the standard (yet)?